lib/requirements_config.yaml

```diff
@@ -0,0 +1,12 @@
+test_kit_id: bulk-data-test-kit
+
+suites: 
+  - id: bulk_data_test_kit
+    class_name: BulkDataTestKit::BulkDataV200::BulkDataTestSuite
+    suite_actor: Provider
+
+requirement_sets:
+  - id: hl7.fhir.uv.bulkdata_2.0.0_reqs
+    actor_map:
+      - {spec: Client, test_kit: Server}
```

Comment:
This actor map will cause Client actor requirements to be associated with the Server. I don't think that's what we want. In this case, there probably aren't any actor changes that you need to make, so you remove that property

Reply:
I got a strange error when removing the actor_map property but got around it by setting it equal to `- { spec: Server, test_kit: Server }`. 

Not sure if necessary but the error was:

  `rake aborted!
  NoMethodError: undefined method `each' for nil:NilClass
  
            req_set_config['actor_map'].each do |actor_mapping|
                                       ^^^^^
  /bulk-data-test-kit/lib/inferno_requirements_tools/tasks/collect_requirements.rb:81:in `block in parse_requirement_set'`

Reply:
That error is saying I didn't have safe navigation on that hash entry.  I updated the code in the base repo to `req_set_config['actor_map']&.each`, which should fix that in the future. Your fix is right for now.